hugo/layouts/news-and-press/single.html

```diff
@@ -8,15 +8,14 @@
 <div class="text-left pb-5">
     <br><br>
     <h1 class="title text-left p-0 mt-5 mb-4">News, Press & Analyst Relations at source{d}</h1>
-    <h2 class="text-left h4 text-light mb-5">Explore the latest news, press coverage and analyst reports about source{d}</h2>
 </div>
 </div>
 </header>
 
 <section id="releases" class="text-dark pb-5">
     <div class="container pt-5">
         <div class="row">
-            <script src='http://www.globenewswire.com/Scripts/gnw_widget.js' type='text/javascript'></script> <script type='text/javascript'> var userActivityUrl ='https://www.globenewswire.com/SocialMediaActivity/Log/jsWidgetRefreshed/50522'; GetWidget('http://www.globenewswire.com/JsonData/organization/9HOlujazceMq1zUGLRgBfA%3d%3d?callback=?', 'Press Releases'); </script><div class='widget__width-height--container'> <div id='gnw_widget' class='gnw_content_container'></div></div> <div class='hideme'></div><div id='hidePbDate' class='hidePbDate'></div>
+            <script src='https://www.globenewswire.com/Scripts/gnw_widget.js' type='text/javascript'></script> <script type='text/javascript'> var userActivityUrl ='https://www.globenewswire.com/SocialMediaActivity/Log/jsWidgetRefreshed/50522'; GetWidget('http://www.globenewswire.com/JsonData/organization/9HOlujazceMq1zUGLRgBfA%3d%3d?callback=?', 'Press Releases'); </script><div class='widget__width-height--container'> <div id='gnw_widget' class='gnw_content_container'></div></div> <div class='hideme'></div><div id='hidePbDate' class='hidePbDate'></div>
```

Comment:
Still not working.
I think it might be because there is an `http` call in `GetWidget('http://www.globenewswire.com/JsonData...` that should be changed by `GetWidget('https://www.globenewswire.com/JsonData...`